pandas/core/computation/expr.py

```diff
@@ -364,7 +364,7 @@ class BaseExprVisitor(ast.NodeVisitor):
 
     unary_ops = _unary_ops_syms
     unary_op_nodes = "UAdd", "USub", "Invert", "Not"
-    unary_op_nodes_map = dict(zip(unary_ops, unary_op_nodes))
+    unary_op_nodes_map = {k: v for k, v in zip(unary_ops, unary_op_nodes)}
```

Comment:
mypy giving 

error: Need type annotation for 'unary_op_nodes_map' (hint: "unary_op_nodes_map: Dict[<type>, <type>] = ...")  [var-annotated]

mypy is ok with a dictcomp, or could add Dict[str, str] as variable type annotation instead